sdk/communication/azure-communication-identity/tests.yml

```diff
@@ -5,18 +5,13 @@ extends:
   parameters:
     PackageName: azure-communication-identity
     SafeName: azurecommunicationidentity
-    Clouds: 'Public,PPE,Int'
+    Clouds: 'Public'
+    UseFederatedAuth: true
     CloudConfig:
       Public:
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePaths:
+          - eng/common/TestResources/sub-config/AzurePublicMsft.json
         SubscriptionConfigurations:
-          - $(sub-config-azure-cloud-test-resources)
           - $(sub-config-communication-services-cloud-test-resources-common)
```

Comment:
Can we use the new secret we created just for CTE here as well and remove the other ones ?

Reply:
Made the names consistent across all SDKs. Hence removed CTE and reusing the old KV.